addons/account/views/report_invoice.xml

```diff
@@ -283,7 +283,9 @@
                                 </div>
                                 <!--terms and conditions-->
                                 <div class="text-muted mb-3" t-attf-style="#{'text-align:justify;text-justify:inter-word;' if o.company_id.terms_type != 'html' else ''}" t-if="not is_html_empty(o.narration)" name="comment">
-                                    <span t-field="o.narration"/>
+                                    <div class="row w-100">
```

Comment:
why not add it to the element? eg:

```suggestion
                                <div class="text-muted mb-3 row w-100" t-attf-style="#{'text-align:justify;text-justify:inter-word;' if o.company_id.terms_type != 'html' else ''}" t-if="not is_html_empty(o.narration)" name="comment">
```
also instead of adding these class, I guess an alternative could be to set `clear:right` in the style attribute, eg:

```suggestion
                                <div class="text-muted mb-3" t-attf-style="clear: right; #{'text-align:justify;text-justify:inter-word;' if o.company_id.terms_type != 'html' else ''}" t-if="not is_html_empty(o.narration)" name="comment">
```

Reply:
Thank you, I will go with  ``clear: right;``